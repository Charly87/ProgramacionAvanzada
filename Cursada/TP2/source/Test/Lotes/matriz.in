4
0 0 94.24735009096824
0 1 17.280661137224385
0 2 63.08671474947168
0 3 9.743292290876438
1 0 21.935990024062058
1 1 14.353802802757043
1 2 66.17438276638433
1 3 34.13920283075178
2 0 46.619215066784605
2 1 29.278632590950536
2 2 70.80631630227869
2 3 17.610555915911963
3 0 23.836834603623192
3 1 38.60441477430502
3 2 98.56406981744567
3 3 99.9159646240917
40.616503793128054
67.93077787039809
7.600938743246188
34.11043116739848
