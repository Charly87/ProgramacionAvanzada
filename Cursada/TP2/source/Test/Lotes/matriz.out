4
-0.0688515002201388
-4.356947310455476
1.9169323632575646
0.15021001780845178

1.4210854715202004E-14
